testutils/check-baselines.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+set -e
+
+# TODO: We should figure out how to do this as part of GapicGeneratorTest at some point.
+
+exitStatus=0
+
+## Check for unhandled cases.
+unhandledStr='$unhandledCallingForm'
+if grep --include='*.baseline' -r -l -F $unhandledStr src/; then
+	cat 1>&2 <<EOF
+Above baseline files contain string "$unhandledStr";
+some sample code won't render properly.
+
+EOF
+	exitStatus=1
+fi
+
+## Check that every calling form is used somewhere.
+callingFormsFile="src/main/java/com/google/api/codegen/viewmodel/CallingForm.java"
+
+# Calling forms are in an enum, so we look for capital identifiers that begins the line.
+callingForms=$(< $callingFormsFile grep -o '^[[:space:]]*[A-Z][a-zA-Z]*' | tr -d -c '[a-zA-Z\n]')
+
+unusedForms=$(
+	# Grep src/
+	#   recursively (-r)
+	#   in baseline files (--include='*.baseline')
+	#   for any calling forms: -f <(cat <<< $callingForms)
```

Comment:
The process substitution is fine and clever, but I'm just curious: any particular reason for  choosing `-f` rather than `-e`?
`grep ... -F $(echo $callingForms | sed 's/[^ ]* */-e &/g') src/`

The command line is similarly (not identically) long in both cases....

Reply:
If we have a lot of forms, splitting with `-e` might blow up the command line enough that the syscall won't execute. With proc substitution, the command line length is the same no matter how many forms we have.

Reply:
We changed to `echo` to fix the issue around newlines in Travis (I should learn more about herestrings) so my objection to the change is mostly moot.

For information though: I think it's different. IIUC, expanding the string in bash is fine, but if the string gets too long, the kernel might not be able to start the process with so many args. By turning into a herestring, the long string is fed into stdin. The stdin gets big but that's OK; the process can still start.